guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,478 @@
 {
```

Comment:
<p>Enhances <code><a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#pspnetclassifier" target="_blank">UnetClassifier</a></code><a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#pspnetclassifier" target="_blank">](* </a><code><a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#unetclassifier" target="_blank">Unet</a></code>) and [<code>PSPNetClassifier</code> models by adding: -&gt; appears to have issues with parenthesis</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/684/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Thanks. Fixed it!</p>